idseq_dag/steps/run_alignment_remotely.py

```diff
@@ -7,6 +6,9 @@
 import time
 import shlex
 
+import os
+from os import path
```

Comment:
path is so often tempting to use as a local variable name, which would lead to aliasing